packages/perseus-core/src/widgets/expression.ts

```diff
@@ -0,0 +1,36 @@
+import type {PerseusExpressionWidgetOptions} from "../data-schema";
+
+export const currentVersion = {major: 1, minor: 0};
+
+export const widgetOptionsUpgrades = {
+    /* c8 ignore next */
```

Comment:
Let's drop this comment.  It's from the original stabilization where we were focused on reaching a certain percentage of test coverage and didn't plan to include this upgrade logic in that coverage calculation.